SPECS/kernel/config_aarch64

```diff
@@ -3387,7 +3387,6 @@ CONFIG_DWMAC_ROCKCHIP=m
 CONFIG_DWMAC_SUN8I=m
 CONFIG_DWMAC_IMX8=m
 # CONFIG_DWMAC_INTEL_PLAT is not set
-# CONFIG_DWMAC_LOONGSON is not set
```

Comment:
MACH_LOONGSON64 is not supported in Marienr 2.0 and a recent changes requires this option be present for the "CONFIG_DWMAC_LOONGSON" config to be available.

https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b7d82e5dbe5a10f2df4f7ac6c015fd6b104d4a94